homework.md

```diff
@@ -44,61 +44,61 @@ Using CSS, how would I select every `<h1>` tag on the page?
 
 What is JavaScript?
 
-A programming language used on the internet
+A programming language sd for aps, games, websites.
 
 ## Question 8
 
 What three problem does Git & Github solve?
 
-When people want to show off code to each other they can put it on Github
+collaboration,  make changes to other people code and merge the code, platform for viewing repositories.
 
 ## Question 9
 
 What happens when you `fork` a repository?
 
-You delete it
+You take a repository from one account to another.
 
 ## Question 10 
 
 What happens when you clone a repostory?
 
-It send it to a friend
+it goes onto your system.
 
 ## Question 11
 
 What is a Pull Request?
 
-When you send a file over the internet
+telling others about changes you have mde to a repo.
 
 ## Question 12
 
 What is the Git History log?
 
-It lets you see into the future
+It lets you see what changs have been made and when.
 
 ## Question 13
 
 What does it meant to "push" changes to Github?
 
-You send the changes of your local repository to Github
+makes the changes you have made accessible to others.
 
 ## Question 14
 
 What is Github?
 
-A place for us to store and share our code.
+Provider of interet hsing for software development
 
 ## Question 15
 
 What does it mean to "Commit your changes"?
 
-Send you code to Github
+Save you code to Github
```

Comment:
It means, to save the changes made to GitHub/desktop.

Reply:
Yes saves the code to Github 👍